backend/src/apiserver/server/visualization_server.go

```diff
@@ -74,5 +74,5 @@ func (s *VisualizationServer) generateVisualizationFromRequest(request *go_clien
 }
 
 func NewVisualizationServer(resourceManager *resource.ResourceManager) *VisualizationServer {
-	return &VisualizationServer{resourceManager: resourceManager, serviceURL: "http://visualization-service.kubeflow"}
+	return &VisualizationServer{resourceManager: resourceManager, serviceURL: "http://ml-pipeline-visualization.kubeflow"}
```

Comment:
let's use const for host. and ".kubeflow" should be the namespace passed from the argument.